### ISSUE TITLE
Process of evaluation and moving between tiers

### ISSUE THREAD

**Issue by glennawatson** [dotnet-foundation/project-maturity-model#37]
We may want to expand the policy to include how the evaluation process for moving between tiers will work and also deciding on initial tiers.

From the process how we evaluation will happen, which groups do the evaluation, and how the report is done back to the participating projects.

**Comment by glennawatson** [comment]
There's some  existing text

```
The working group will process ladder registration requests regularly. The same model will be used for new projects as well as level promotions. One of the following approaches will be used:

* The working group accepts the registration, possibly with requested improvements.
* The working group rejects the registration, with feedback.
* The working group puts the registration up for a membership vote, and makes a public decision based on the result of the vote. This is more likely for level 3 and 4 candidates.

Ladder level promotions need to be championed by a working group member who agrees to mentor the library, or identifies a community member who can provide mentorship.
```

This doesn't describe the process for deciding on how projects move between tiers.

**Comment by glennawatson** [comment]
I've addressed this in #38
